src/view/view.ts

```diff
@@ -214,10 +214,16 @@ export class TrelloView extends ItemView {
   private renderCardTitle(card: TrelloCard, parent: HTMLElement): void {
     this.plugin.log('TrelloView.renderCardTitle', '');
     const cardName = parent.createEl('h3', { text: card.name });
-    const cardLink = cardName.createEl('a', {
-      attr: { href: card.url, 'aria-label': 'View on Trello' }
+
+    let cardLink = String(card.url);
```

Comment:
Isn't `card.url` already a string here? I don't think this cast is necessary.
```suggestion
    let cardLink = card.url;
```

Reply:
@nathonius I can't remember, but I think it sometimes comes back as a URL? But there's no warnings in VScode, so I guess it's OK 🤞